### PR TITLE
Fixes the setproctitle bug, and the wheel bug.

### PR DESCRIPTION
**Pull request by richardkiss**
We've forked this a Chia <https://chia.net/> to make some fixes for Mac OS... can you take a look and merge or let us know how to improve this fix? Thank you.

### CODE REVIEW

**Comment by richardkiss** [comment]
@mariano54

File: src/spt_setup.c [hunk 1]
```diff
@@ -447,7 +467,7 @@ spt_setup(void)
 #ifndef WIN32
     int argc = 0;
     char **argv = NULL;
-    char *init_title;
+    char *init_title = "chia";
```
Comment (dvarrazzo) [inline src/spt_setup.c]: what?

File: src/spt_python.h [hunk 2]
```diff
@@ -14,9 +14,7 @@
 #include <Python.h>
 
 /* Things change a lot here... */
-#if PY_MAJOR_VERSION >= 3
```
Comment (dvarrazzo) [inline src/spt_python.h]: Py2 is not dropped yet until the next major release. Semver, you know

File: src/spt_setup.c [hunk 3]
```diff
@@ -369,8 +377,10 @@ get_argc_argv(int *argc_o, char ***argv_o)
     int rv = -1;
 
 #ifndef IS_PYPY
+#ifndef __APPLE__
```
Comment (dvarrazzo) [inline src/spt_setup.c]: You haven't really explained the rationale of this change. What doesn't work? Why it works this way?

**Comment by richardkiss** [comment]
@mariano54 made these changes (for our internal fork, you can blame me for the pull request), maybe he can speak to what motivated each of these change. Apologies for the half-bake pull request, I'm sort of just trying to motivate a merge which I believe fixes some problems on Mac OS X 10.15.

**Comment by dvarrazzo** [comment]
Ok closing this now then. When you have a proper MR let me know.

I don't have an iOS to check, so please create a test on travis showing the error without the patch and the clean build afterwards.

**Pull request closed without merging**